doc/whats_new.rst

```diff
@@ -153,11 +153,17 @@ Enhancements
 Bug fixes
 .........
 
+   - Fixed a bug where :func:`sklearn.model_selection.BaseSearchCV.inverse_transform`
+     returns self.best_estimator_.transform() instead of self.best_estimator_.inverse_transform()
+     :issue:`8344` by :user:`Akshay Gupta <Akshay0724>` 
+
+
    - Fixed a bug where :class:`sklearn.linear_model.RandomizedLasso` and
      :class:`sklearn.linear_model.RandomizedLogisticRegression` breaks for
      sparse input.
      :issue:`8259` by :user:`Aman Dalmia <dalmia>`.
 
+
```

Comment:
No problem, but it's always good to review your diff before and/or after commit to reduce diff noise like this.